mingw-bundledlls

```diff
@@ -33,6 +33,10 @@ default_path_prefixes = [
     "", "/usr/bin", "/usr/i686-w64-mingw32/sys-root/mingw/bin", "/mingw64/bin",
 ]
 
+env_path_prefixes = os.environ.get('MINGW_BUNDLEDLLS_SEARCH_PATH', None)
+if env_path_prefixes is not None:
+    default_path_prefixes = filter(None, env_path_prefixes.split(os.pathsep))
```

Comment:
Please rename the variable `default_path_prefixes` to just `path_prefixes` or something like that since you changed the meaning of it. And should the filter statement be `filter(None.__ne__, ...)`? Other than that I like the patch :-)